genie-agent/src/test/resources/JobSetupServiceImplSpec_run1.test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#
+# Generated by Genie for job: <JOB_ID_PLACEHOLDER>
+#
+
+# Error out if any command fails
+set -o errexit
+# Error out if any command in a pipeline fails
+set -o pipefail
+# Error out if unknown variable is used
+set -o nounset
+
+
+export GENIE_JOB_DIR="<JOB_DIR_PLACEHOLDER>"
+
+export GENIE_APPLICATION_DIR="${GENIE_JOB_DIR}/genie/applications"
+
+export GENIE_COMMAND_DIR="${GENIE_JOB_DIR}/genie/command/<COMMAND_ID_PLACEHOLDER>"
+
+export GENIE_CLUSTER_DIR="${GENIE_JOB_DIR}/genie/cluster/<CLUSTER_ID_PLACEHOLDER>"
+
+export __GENIE_SETUP_LOG_FILE="${GENIE_JOB_DIR}/genie/logs/setup.log"
```

Comment:
any reason to name these specially with the `__`?

Reply:
My own notation for environment variables that are private in the script.